test/gtest/ucp/test_ucp_rma.cc

```diff
@@ -182,7 +182,7 @@ class test_ucp_rma : public test_ucp_memheap {
     }
 
     bool disable_proto() {
-        return !m_ucp_config->ctx.proto_enable;
+        return !is_proto_enabled();
```

Comment:
I guess it's a bug here (from this PR https://github.com/openucx/ucx/pull/9084).
This method should check test variant value, and the config value should be set based on return value.
Thus, it should be:
 ```C
 bool disable_proto() const
{
        return get_variant_value() & DISABLE_PROTO;
}
```

And then `!is_proto_enabled()` should be used in this file at these lines: 295 and 314.

Reply:
config maybe set externally (e.g. in proto_disable roce tests in CI)

Reply:
I see.
But we don't use `DISABLE_PROTO` variant value in these tests.